src/libs/datatableproviders/DataProvider.ts

```diff
@@ -0,0 +1,78 @@
+// define metadata structures
+interface EntityTypeMetadata {
+    attributeNames: string[],
+    count: number,
+    idName: string
+}
+
+export interface EntityMetadata {
+    [index: string]: EntityTypeMetadata
+}
+
+export enum EntityQuerySortDirection {
+    // eslint-disable-next-line no-unused-vars
+    Asc = 'asc',
+    // eslint-disable-next-line no-unused-vars
+    Desc = 'desc'
+}
+
+export enum EntityQueryFilterOperator {
+    // eslint-disable-next-line no-unused-vars
+    And = 'and',
+    // eslint-disable-next-line no-unused-vars
+    Or = 'or'
+}
```

Comment:
even though I use these enums inside `EntityQuery`, the individual enum values complain about being unused.

Reply:
It looks like these disable comments are no longer necessary after switching to the `@typescript-eslint/no-unused-vars` ESLint rule.